src/controlles/file-controller.js

```
@@ -52,7 +43,7 @@ const lastUpdateTime = async (filePath) => {
 };
 
 const errorHandlingFunc = (err, res) => {
-    res.write(err.toString());
+    res.write('internal server error');
```

when you are in production mode (NODE_ENV==='production') return 'internal server error' but in develop its better to return the error itself